src/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  background-color: #fff;
+}
+
+h1 {
+  display: none;
+}
+
+.to-do-list {
+  margin: 5% 25%;
+  padding: 0;
```

Comment:
- [ ]  Please use best practice CSS, I do see a lot of repeating rules (padding, font size, color, and bored) also adjust the font size to 1rem.